tests/test_req_relaxed.cpp

```diff
@@ -169,7 +169,7 @@ void test_case_4 ()
     TEST_ASSERT_SUCCESS_ERRNO (
       zmq_setsockopt (req, ZMQ_REQ_CORRELATE, &enabled, sizeof (int)));
 
-    TEST_ASSERT_SUCCESS_ERRNO (zmq_connect (req, ENDPOINT_0));
+    TEST_ASSERT_SUCCESS_ERRNO (zmq_connect (req, ENDPOINT_7));
```

Comment:
This doesn't exist

Reply:
How does it compile then? In fact, those tests were spuriously failing as ENDPOINT_0 was shared between two unrelated tests, sometimes causing one or the other to randomly fail with an "address in use" error. This is a fix.

Reply:
Oh I see the change to testutil.hpp did not make it. Fixed.